Documentation/network/servicemesh/mutual-authentication/installation.rst

```diff
@@ -0,0 +1,58 @@
+Prerequisites
+#############
+
+* Mutual authentication is only currently supported with SPIFFE APIs for certificate management. 
+* Users have the option to install a SPIRE server or they can also bring their own instead.
+  
+Installation
+############
+
+.. tabs::
+
+    .. group-tab:: Helm
+
+        Mutual authentication can be enabled with helm flag ``authentication.mutual.spire.enabled``
+        set as true. 
+        A SPIRE server can be deployed alongside it with the helm
+        flag ``authenticaton.mutual.spire.install.enabled`` also set as true.
+        Please refer to :ref:`k8s_install_helm` for a fresh installation.
+
+        .. parsed-literal::
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace kube-system \\
+                --reuse-values \\
+                --set authentication.mutual.spire.enabled=true \\
+                --set authentication.mutual.spire.install.enabled=true
+
+            $ kubectl -n kube-system rollout restart deployment/cilium-operator
+            $ kubectl -n kube-system rollout restart ds/cilium
+
+        
+        Next you can check the status of the Cilium agent and operator:
+
+        .. code-block:: shell-session
+
+            $ cilium status
+
+        .. include:: ../../../installation/cli-download.rst
+
+    .. group-tab:: Cilium CLI
+
+        .. include:: ../../../installation/cli-download.rst
+
+        Mutual authentication and associated SPIRE server can be enabled with the following command. 
+        Note this requires the Cilium CLI Helm mode (enabled with setting the environment variable with ``CILIUM_CLI_MODE=helm``).
+
+        .. code-block:: shell-session
+
+            $ cilium install \\
+                --helm-set authentication.mutual.spire.enabled=true \\
```

Comment:
You need the double backslashes in the `.. parse-literal::` blocks, because backslashes are parsed as RST escape markers there, but I think you need just one per line in the `.. code-block::`. Please double-check on the preview.

```suggestion
            $ cilium install \
                --helm-set authentication.mutual.spire.enabled=true \
```